sdk/storage/azure-storage-common/src/storage_exception.cpp

```diff
@@ -141,6 +141,12 @@ namespace Azure { namespace Storage {
       }
     }
 
+    if (errorCode.empty()
```

Comment:
Will this catch all such cases across all the storage libraries or do we have some other paths that need similar fixes?

Reply:
This is common package. It'll cover all storage packages.